workbench-core/example/app/src/configs/staticPermissionsConfig.ts

```diff
@@ -31,6 +31,21 @@ const adminPermissions: Permission[] = [
     action: 'DELETE',
     subject: 'Dataset'
   },
+  {
+    effect: 'ALLOW',
+    action: 'READ',
+    subject: 'Storage'
+  },
```

Comment:
Is this from your listStorageLocations integ tests work?

Reply:
yes